### ISSUE TITLE
Failed to show json message with packets, using command approach

### ISSUE THREAD

**Issue by PhracKko** [Zrips/CMI#3588]
**Description of issue:**

Packet error occurs when using /cmi tps command

(just when use in client, not in a console )
---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
######################
## ##
######################
```

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
#######################################
## REPLACE WITH RELEVANT CONFIG SECTION  ##
#######################################
```
[13:17:22 INFO]: UBASE issued server command: /cmi tps
[13:17:22 WARN]: Caused by: com.google.gson.JsonParseException: Unexpected empty array of components
[13:17:22 WARN]:        ... 28 more
[13:17:22 INFO]: [CMI] Failed to show json message with packets, using command approach
[13:17:22 INFO]: ...w "UBASE" ["",{"text":"","extra":[{"text":"TPS: "}]},{"text":"","extra":[{"text":""}]},{"text":"","extra":[{"text":"19.96"}],"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"3 초 Max: 20.0 Min: 20.0"}]}}},{"text":"","extra":[ ]},{"text":"","extra":[{"text":"19.98"}],"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"10 초 Max: 20.0 Min: 19.96"}]}}},{"text":"","extra":[ ]},{"text":"","extra":[{"text":"19.99"}],"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"30 초 Max: 20.0 Min: 19.96"}]}}},{"text":"","extra":[ ]},{"text":"","extra":[ ]}]<--[HERE]
---

**Cmi Version (using`/cmi version`):** 
  latest (8.7.0.2)

**Server Type (Spigot/Paperspigot/etc):** 
[13:18:21 INFO]: This server is running Paper version git-Paper-372 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[13:18:21 INFO]: Checking version, please wait...
[13:18:23 INFO]: Previous version: git-Paper-354 (MC: 1.15.2)

**Server Version (using `/ver`):** 
 ( "--" )
**Relevant plugins (Delete if this isn't needed):**
`<img src="https://phrack.xyz/data/file/free/2728312410_PhxcZNTB_a062744d77b7da4d28841082727547e1983fd364.png" style="width:auto;"`>
![123](https://user-images.githubusercontent.com/61553558/85937878-9f4fab00-b942-11ea-93a6-98285ad31af2.png)

**Comment by PhracKko** [comment]
[00:06:19 WARN]: java.lang.reflect.InvocationTargetException
[00:06:19 WARN]:        at sun.reflect.GeneratedMethodAccessor78.invoke(Unknown Source)
[00:06:19 WARN]:        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[00:06:19 WARN]:        at java.lang.reflect.Method.invoke(Unknown Source)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.RawMessages.RawMessageManager.send(RawMessageManager.java:109)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.RawMessages.RawMessage.show(RawMessage.java:689)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.RawMessages.RawMessage.show(RawMessage.java:673)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.RawMessages.RawMessage.show(RawMessage.java:719)
[00:06:19 WARN]:        at com.Zrips.CMI.commands.list.tps.perform(tps.java:87)
[00:06:19 WARN]:        at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:323)
[00:06:19 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.dispatch(SpecializedCommandManager.java:757)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.executeCmd(SpecializedCommandManager.java:624)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.processCmds(SpecializedCommandManager.java:144)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.processCmds(SpecializedCommandManager.java:112)
[00:06:19 WARN]:        at com.Zrips.CMI.Modules.InteractiveCommand.InteractiveCommandListener$1.run(InteractiveCommandListener.java:106)
[00:06:19 WARN]:        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:99)
[00:06:19 WARN]:        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.MinecraftServer.b(MinecraftServer.java:1264)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:431)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1181)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:970)
[00:06:19 WARN]:        at java.lang.Thread.run(Unknown Source)
[00:06:19 WARN]: Caused by: com.google.gson.JsonParseException: Unexpected empty array of components
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.IChatBaseComponent$ChatSerializer.deserialize(IChatBaseComponent.java:315)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.IChatBaseComponent$ChatSerializer.deserialize(IChatBaseComponent.java:231)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.IChatBaseComponent$ChatSerializer.deserialize(IChatBaseComponent.java:184)
[00:06:19 WARN]:        at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.ChatDeserializer.a(SourceFile:494)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.ChatDeserializer.a(SourceFile:519)
[00:06:19 WARN]:        at net.minecraft.server.v1_15_R1.IChatBaseComponent$ChatSerializer.a(IChatBaseComponent.java:445)
[00:06:19 WARN]:        ... 22 more

**Comment by PhracKko** [comment]
fixed! at 8.7.0.3

**Comment by Zrips** [comment]
Fixed
